File: htdocs/thedatalakeap/login_admin/insert_usr.php
                         <?php 
                           $usrid = ($_POST['id']);
                           $pass = sha1($_POST['pass']);
                           $message= "User inserido com sucesso";
                          

                           $ligax = mysqli_connect('localhost', 'root', '');
                           if (!$ligax)  {echo '<p> Erro: Falha na ligação.'; exit;};
                           mysqli_select_db($ligax, 'thedatalakeap');
                           $insere = "INSERT INTO users ( usr_id, password ) VALUES ('".$usrid."', '".$pass."')";
                           $result = mysqli_query($ligax, $insere);
                           if ($result==1){



header("Location:usr_inserted.html");
//echo '<script type="text/javascript">';
//echo '<script type=\"text/javascript\">alert("O registo foi inserido na Base de Dados");</script>';

//echo '</script>' ;
		// o utilizador está correctamente validado
		// guardamos as suas informações numa sessão
		//$_SESSION['id'] = mysqli_result($login, 0, 0);
		//$_SESSION['usrid'] = mysqli_result($login, 0, 1);

       //  echo "<script type='text/javascript'>alert('Utilizador inserido com sucesso')</script>";
        
//echo '<script language="javascript">';
//echo 'alert("message successfully sent")';
//echo '</script>';
  







	} else {
 
		// falhou o login
         
header("Location:usr_not_inserted.html");
        
         
	}

                        ?>

File: htdocs/thedatalakeap/login_admin/insert_usr.py
#!\Python27\python.exe
import mysql.connector

mydb = mysql.connector.connect(
  host="localhost",
  user="root",
  passwd="",
  database="thedatalakeap"
)
def signup():
    cur = g.db.cursor()
    id_user = request.form['id']
    password = sha1(request.form['pass'])
    cur.execute("INSERT INTO users (usr_id, password) VALUES (?, ?)", [id_user, password])
    #cur.execute("""SELECT email,username FROM users WHERE email=? OR username=?""",(email, username))
    result = cur.fetchone()
  
    if result:

        return redirect('/')
 

    else:
        cur.execute("INSERT INTO users VALUES (?, ?, ?)", [email, username, password])
        g.db.commit()
        return redirect('/')
mydb.close()